src/utilities.js

```diff
@@ -2,9 +2,25 @@ import Cookie from 'universal-cookie';
 
 import { DEFAULT_IETF_TAG, LANGUAGE_CODES } from './constants';
 
+const isStage = () => {
+  const host = window.location.hostname;
+
+  // edx.org uses different subdomains so use a root domain to match across services
+  return host.indexOf('stage.edx.org') !== -1 ||
```

Comment:
You could consider moving all these domains to an array and then iterating over that array

```javascript
const STAGE_SUBDOMAINS = Object.freeze(['stage.edx.org', 'dev.edx.org', 'acceptance.edx.org', 'qa.edx.org']);

const isStage = () => STAGE_SUBDOMAINS.every(subdomain => window.location.hostname.indexOf(subdomain) >= 0);
```

Looks like [`every`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) is compatible with `IE9`

![image](https://user-images.githubusercontent.com/8136030/38577749-d2892218-3ccf-11e8-96c2-375534142dd8.png)

Reply:
Sorry, my previous suggestion was incorrect.

`every` wouldn't work, because obviously the `hostname` is not going to contain `stage` and `dev` and `acceptance`, etc. so it'll always be `false`.

Really, what we want is an "any" method (there's [`find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) but it's not `IE`-compatible).

If you want to continue with the array-based approach you could instead use [`filter`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) like

```javascript
const isStage = () => STAGE_SUBDOMAINS.filter(
  subdomain => window.location.hostname.indexOf(subdomain) >= 0
).length > 0;
```

![image](https://user-images.githubusercontent.com/8136030/38581696-98a5beb0-3cdb-11e8-8b34-f9cb1c46c1e1.png)